code/modules/surgery/bodyparts/dismemberment.dm

```diff
@@ -10,7 +10,7 @@
 	var/mob/living/carbon/C = owner
 	if(C.status_flags & GODMODE)
 		return FALSE
-	if(HAS_TRAIT(C, TRAIT_NODISMEMBER))
+	if(HAS_TRAIT(C, TRAIT_NODISMEMBER) && !HAS_TRAIT_FROM(C, TRAIT_EASYDISMEMBER, SPECIES_TRAIT))
```

Comment:
Choose one trait to gain precedence over the other and document it in the definition in `traits.dm`, please.
In this case it makes more sense for no-dismember to have precedence over easy-dismember, so you'd do better by having a bypass-no-dismember if you don't want to refactor this (ideally, since this is a complex equation, dismemberability wouldn't be a binary value).

What we don't want, however, is to have a system where no-dismember trumps easy-dismember, but there's this little exception for species easy-dismember in where it behaves the other way. It needs to be consistent.